### PR TITLE
Secure your wallet page

### LINKED ISSUE #13446: Secure your wallet page
- We should not vertically center the bulleted list. 

Figma file for reference: https://www.figma.com/file/iCOjL3k1BkQqpYHFUVVFUZ/Extension-Onboarding?node-id=1525%3A15961

needs design feedback:
cc @rachelcope

### PR DESCRIPTION
**Pull request by VSaric**
fixes: #13446

**Explanation:**  Secure your wallet page

**Screencast:**

https://user-images.githubusercontent.com/92527393/153590156-7060f62b-4f54-4193-852b-230a1d9893a8.mov

### CODE REVIEW

**Comment by mirjanaKukic** [comment]
Verified by QA.

**Review by brad-decker — APPROVED** [review]

File: ui/pages/onboarding-flow/secure-your-wallet/index.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   &__details {
     max-width: 550px;
+    padding-right: 10px;
```
Comment (darkwing) [inline ui/pages/onboarding-flow/secure-your-wallet/index.scss:11]: `padding-inline-end` would be better here for RTL support.

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js [hunk 2]
```diff
@@ -121,56 +118,53 @@ export default function SecureYourWallet() {
...
-        <Typography tag="span" variant={TYPOGRAPHY.H4}>
-          {t('seedPhraseIntroSidebarCopyThree')}
-        </Typography>
+      <Box>
+        <Box marginBottom={4}>
+          <Typography
+            tag="span"
+            variant={TYPOGRAPHY.H4}
+            fontWeight={FONT_WEIGHT.BOLD}
+            boxProps={{ display: 'block' }}
```
Comment (darkwing) [inline ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:127]: If we're going to `display: block` this, we likely don't need a `tag="span"`.

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js [hunk 3]
```diff
@@ -121,56 +118,53 @@ export default function SecureYourWallet() {
...
+          <Typography tag="span" variant={TYPOGRAPHY.H4}>
+            {t('seedPhraseIntroSidebarCopyOne')}
+          </Typography>
+        </Box>
+        <Box marginBottom={4}>
+          <Typography
+            tag="span"
+            variant={TYPOGRAPHY.H4}
+            fontWeight={FONT_WEIGHT.BOLD}
+            boxProps={{ display: 'block' }}
```
Comment (darkwing) [inline ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:140]: Same here; can this just be a `tag="div"`, or `tag="p"`?

**Review by darkwing — DISMISSED** [review]

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js (hunk 3 above)
Comment (brad-decker) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:140]: not div, for text contents. p is more appropriate.

Comment (brad-decker) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js:140]: Also i'm now catching we aren't using the DISPLAY constants for these from the design-system file. :P [req]

**Review by brad-decker — CHANGES_REQUESTED** [review]

File: ui/pages/onboarding-flow/secure-your-wallet/index.scss (hunk 1 above)
Comment (VSaric) [reply ui/pages/onboarding-flow/secure-your-wallet/index.scss]: @darkwing Fixed in: 58720e7d4f52a70703f7d7f6502b31dcbe7493b3

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js (hunk 2 above)
Comment (VSaric) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js]: @darkwing I changed `tag="span"` to `tag="p"` as @brad-decker suggested.
Fixed in: 58720e7d4f52a70703f7d7f6502b31dcbe7493b3

File: ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js (hunk 3 above)
Comment (VSaric) [reply ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js]: @darkwing @brad-decker I changed `tag="span"` to `tag="p"` and use `DISPLAY` constant from the design-system file.
Fixed in: 58720e7d4f52a70703f7d7f6502b31dcbe7493b3

**Review by brad-decker — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**